pkg/bundle/bundle.go

```diff
@@ -35,12 +35,12 @@ func PackAndPush(ref name.Reference, directory string, options ...remote.Option)
 		return name.Digest{}, err
 	}
 
-	image, err := mutate.AppendLayers(empty.Image, bundleLayer)
+	image, err := mutate.Time(empty.Image, time.Unix(0, 0))
```

Comment:
Mutating the empty image before using it with `AppendLayers` makes sure that the image has timestamp 0, but the layer that is later added is not mutated as well.